runtime-decompiler/src/main/java/org/jrd/frontend/frame/main/MainFrameView.java

```diff
@@ -313,6 +315,11 @@ public void mouseClicked(MouseEvent mouseEvent) {
             pluginConfigurationEditorListener.actionPerformed(actionEvent);
         });
         jMenuConfig.add(jMenuPluginEditor);
+        jMenuNestedJars = new JMenuItem("Nested Jars");
```

Comment:
Lets introduce general "setting" menu item here - and put tabbed pane in. NestedJars will be one tab. class path less compiler should be another, and agent setup should be third. For plugin, the separate menu item is still worthy.

Agents have to remian first tab.

Thanx!

Reply:
I would be of the opinion to put of the mentioned settings into what is now called `Configure -> Agent` (should be renamed to `Configure -> Settings`). There is plenty of room in that window, and making all the settings visible at once is a benefit.

Reply:
agent may arrive any time later.